server/norace_test.go

```diff
@@ -1756,6 +1757,89 @@ func TestNoRaceJetStreamClusterSourcesMuxd(t *testing.T) {
 
 }
 
+func TestNoRaceJetStreamClusterExtendedStreamPurgeStall(t *testing.T) {
+	t.Skip("fails always")
+
+	cerr := func(t *testing.T, err error) {
+		t.Helper()
+		if err != nil {
+			t.Fatalf("unexepected err: %s", err)
+		}
+	}
+
+	s := RunBasicJetStreamServer()
+	defer s.Shutdown()
+
+	nc, js := jsClientConnect(t, s)
+	defer nc.Close()
+
+	cfg := StreamConfig{
+		Name:     "KV",
+		Subjects: []string{"kv.>"},
+		Storage:  FileStorage,
+	}
+	req, err := json.Marshal(cfg)
+	cerr(t, err)
+
+	_, err = nc.Request(fmt.Sprintf(JSApiStreamCreateT, cfg.Name), req, time.Second)
+	cerr(t, err)
+
+	si, err := js.StreamInfo("KV")
+	cerr(t, err)
+
+	// 10kb messages spread over 1000 subjects
+	body := make([]byte, 101024)
```

Comment:
This supposed to be 10*1024?

Reply:
something like that :P I probably just fiddled numbers till it broke, should be 100kb